src/utils/stacUtils.tsx

```diff
@@ -89,9 +96,10 @@ export const parseFeatureDataPoints = (feature: StacItem): DataPoint[] => {
 
   if (datetime) {
     dataPoints.push({
+      id: `${feature.collection}_datetime`,
       icon: IoTimeOutline,
       alt: 'Time Icon',
-      value: formatDate(datetime, true),
+      value: formatDate(datetime, TIME_DATE_FORMAT),
```

Comment:
For here, I would have exported the `DEFAULT_DATE_FORMAT` then used it as in 

```
formatDate(datetime, `${DEFAULT_DATE_FORMAT}Thh:mm:ss`);
```

Let me know what you think.